### PR TITLE
Pulled prismic data for Contact Page

### LINKED ISSUE #12: Pull Data: Contact Page

### PR DESCRIPTION
**Pull request by jzhzhang66**
Pulled initial prismic data for Contact page 
Added a function call to retrieve contact page data in api.js (getContactPage)
Contact page

### CODE REVIEW

**Review by josephmannis — APPROVED** [review]
☀️

File: pages/contact.js [hunk 1]
```diff
@@ -3,13 +3,13 @@ import { getContactPage } from "../lib/api";
...
-        contact_page_description,
+        contact_page_title: contactPageTitle,
+        contact_page_description: contactPageDescription,
     } = contactPageData;
 
     return <>
-    {renderRichText(contact_page_title)}
-    {renderRichText(contact_page_description)}
+    {renderRichText(contactPageTitle)}
+    {renderRichText(contactPageDescription)}
```
Comment (isabellberzsenyi) [inline pages/contact.js]: Styling: let's put parentheses after the return around these lines

**Review by isabellberzsenyi — CHANGES_REQUESTED** [review]
Minor styling comment, but ready to approve after

File: pages/contact.js (hunk 1 above)
Comment (jzhzhang66) [reply pages/contact.js]: Thanks! Just fixed and pushed.

**Review by isabellberzsenyi — APPROVED** [review]
✨

**Review by jean-zhang — APPROVED** [review]
LGTM!

**Pull request merged**